ldap.js

```diff
@@ -83,7 +114,22 @@ exports.getUser = (username, cb) => {
       if (!ret.length) {
         return cb(null, null);
       } else {
-        return cb(null, ret[0]);
+        searchGroups(ret[0].uid, function(err, groups) {
+          if(err) {
+            return cb(null, null);
+          } else {
+            var admin = false;
+            for(var i = 0; i < groups.length; i++) {
+              if(groups[i] === "atlas-admin") {
```

Comment:
Use a constant defined at the top of the file, e.g.:

```node
const ATLAS_ADMIN_LDAP_GROUP = "atlas-admin";
```

Magic values (numeric or string) are good to avoid. The moment you come along and need to reference this group name somewhere else in the code, it becomes fragile. A constant can also help clarify the purpose of the value.

Reply:
I made the changes, and everything seems to work as intended. :)